modules/nf-core/smncopynumbercaller/meta.yml

```diff
@@ -0,0 +1,35 @@
+name: "smncopynumbercaller"
+description: tool to call the copy number of full-length SMN1, full-length SMN2, as well as SMN2Δ7–8 (SMN2 with a deletion of Exon7-8) from a whole-genome sequencing (WGS) BAM file.
+keywords:
+  - copy number, BAM, CRAM, SMN1, SMN2
+tools:
+  - "smncopynumbercaller":
+      description: "call copy number of SMN1, SMN2, SMN2Δ7–8 from a bam file"
+      homepage: "https://github.com/Illumina/SMNCopyNumberCaller"
+      documentation: "https://github.com/Illumina/SMNCopyNumberCaller"
+      tool_dev_url: "https://github.com/Illumina/SMNCopyNumberCaller"
+      doi: "https://doi.org/10.1038/s41436-020-0754-0"
+      licence: "Apache License Version 2.0"
+
+input:
```

Comment:
Don't forget to add `meta` and `bai` to the input

Reply:
Thanks for the comments, I will do that.